src/commands/translate/translate.command.js

```diff
@@ -27,6 +26,17 @@ const createTranslationMessage = (from, to, translation) => {
 
 const translator = new Translator();
 
+// it errors when in test env
+// not sure why but _getSupportedLanguages()
+// seems to be overriden by jest or smt
+// (it returns undefined)
```

Comment:
Yeah that makes sense. If you look at `translate.command.spec.js` you'll see it's asking Jest to create a mock version of the `translator` module. So it overrides every class function with a stub version that returns null or undefined or something.

I see what you're trying to achieve here though. It looks like you want the bot to fetch all supported languages at start up so that it doesn't have to make that API call on every suggestion. I really like that idea, I'm glad you thought about it. Sounds efficient.

Here's an idea: instead of running this function at the top level of this module, why not run this function inside the `client.on('ready')` callback in the bot's boot up? that way it wont cause any errors in tests because no one will be calling that function but it will still run once during start up.

Also, it would be nice for the function to be called every once in a while... like maybe once every 24hrs just so that it keeps up with a change in the list of languages and doesn't have to wait until the next restart. You could use a `setInterval` for that